packages/react/src/components/Select/Select.tsx

```diff
@@ -196,54 +199,66 @@ const Select = (props: SelectProps) => {
     multipleChangeHandler([]);
   };
 
-  const moveFocusDown = useCallback(() => {
-    if (activeOption === undefined) {
-      setActiveOption(sortedOptions[0].value);
-    } else {
-      const newIndex = activeOptionIndex + 1;
-      if (newIndex >= 0 && newIndex < numberOfOptions) {
-        setActiveOption(sortedOptions[newIndex].value);
+  const moveFocusDown = useCallback(
+    () => {
+      let newActiveOption = null;
+      if (activeOption === undefined) {
+        newActiveOption = sortedOptions[0];
+      } else {
+        const newIndex = activeOptionIndex + 1;
+        if (newIndex >= 0 && newIndex < numberOfOptions) {
+          newActiveOption = sortedOptions[newIndex];
+        }
       }
-    }
-    setExpanded(true);
-  }, [
-    activeOption,
-    activeOptionIndex,
-    setActiveOption,
-    sortedOptions,
-    numberOfOptions,
-  ]);
-
-  const moveFocusUp = useCallback(() => {
-    if (activeOption === undefined) {
-      setActiveOption(sortedOptions[numberOfOptions - 1].value);
-    } else {
-      const newIndex = activeOptionIndex - 1;
-      if (newIndex >= 0 && newIndex < numberOfOptions) {
-        setActiveOption(sortedOptions[newIndex].value);
+      if (newActiveOption) {
+        setActiveOption(newActiveOption.value);
+        resetKeyword(newActiveOption.label);
       }
-    }
-    setExpanded(true);
-  }, [
-    activeOption,
-    activeOptionIndex,
-    setActiveOption,
-    sortedOptions,
-    numberOfOptions,
-  ]);
+      setExpanded(true);
+    },
+    /* eslint-disable-next-line react-hooks/exhaustive-deps */
```

Comment:
Should we really disable this? This might lead to other errors down the line if this linting is disabled.